channel-jira-server/src/main/java/com/synopsys/integration/alert/channel/jira/server/distribution/delegate/JiraServerIssueCreator.java

```diff
@@ -94,6 +109,19 @@ protected IssueCreationRequestModel createIssueCreationRequest(IssueCreationMode
         );
     }
 
+    @Override
+    protected List<JiraSearcherResponseModel> searchForIssue(IssueCreationModel alertIssueCreationModel) {
+        String query = alertIssueCreationModel.getQueryString().orElse(null);
```

Comment:
Is there potential for a NPE here and line 117? It appears in other places we check if the query is there with StringUtils.isBlank

Reply:
It's not really possible but I will make a change to address it.